helm/frost-server/templates/mqtt-deployment.yaml

```diff
@@ -62,17 +62,17 @@ spec:
             - name: mqtt_mqttServerImplementationClass
               value: "de.fraunhofer.iosb.ilt.sensorthingsserver.mqtt.moquette.MoquetteMqttServer"
             - name: mqtt_Enabled
-              value: "true"
+              value: "{{ .Values.frost.mqtt.enabled }}"
             - name: mqtt_Port
-              value: "1883"
+              value: "{{ .Values.frost.mqtt.ports.mqtt.servicePort }}"
             - name: mqtt_Host
-              value: "0.0.0.0"
```

Comment:
This is incorrect. mqtt_Host is the interfaces the Java program listens on _inside the pod_. This should always be 0.0.0.0 (all interfaces). `Values.frost.mqtt.host` is the hostname outside the cluster.